plugins/woocommerce-admin/client/marketplace/components/header/header.scss

```diff
@@ -7,10 +7,7 @@
 	display: grid;
 	grid-template: 'mktpl-title mktpl-search mktpl-meta' 60px
 		'mktpl-tabs mktpl-tabs mktpl-tabs' auto / 1fr 320px 36px;
-	left: 0;
 	padding: 0 $content-spacing-large;
-	position: absolute;
-	top: -60px;
```

Comment:
No need to push up the header with negative values, as long as we reset margins in `.woocommerce-layout__primary` and `#wpbody`. These are set by the WordPress core and WooCommerce Admin. I did this in `marketplace.scss`.